Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -16,6 +16,27 @@ import UIKit
 import ARKit
 import ArcGIS
 
+extension AGSDeviceOrientation {
+    /// Allows creation of an `AGSDeviceOrientation` from a `UIDeviceOrientation`.
+    ///
+    /// - Parameter statusBarOrientation: The `UIDeviceOrientation` to create the `AGSDeviceOrientation` from.
+    init?(statusBarOrientation: UIDeviceOrientation) {
+        switch statusBarOrientation {
+        case .landscapeLeft:
+            self.init(rawValue: AGSDeviceOrientation.landscapeRight.rawValue)
+        case .landscapeRight:
+            self.init(rawValue: AGSDeviceOrientation.landscapeLeft.rawValue)
```

Comment:
Should this be `landscapeRight`?

Reply:
The SceneView API has them reversed from the UIDeviceOrientation values.